cmd/expiration-mailer/main.go

```diff
@@ -268,9 +270,27 @@ func (m *mailer) findExpiringCertificates() error {
 			continue // nothing to do
 		}
 
+		// If the `certs` result was exactly `m.limit` rows we need to increment
+		// a stat indicating that this nag group is at capacity based on the
+		// configured cert limit. If this condition continually occurs across mailer
+		// runs then we will not catch up, resulting in under-sending expiration
+		// mails. The effects of this were initially described in issue #2002[0].
+		//
+		// 0: https://github.com/letsencrypt/boulder/issues/2002
+		if len(certs) == m.limit {
+			m.log.Info(fmt.Sprintf(
+				"expiration-mailer: nag group %s expiring certificates at configured capacity (cert limit %d)\n",
```

Comment:
I was following the convention of the other log statements in the file. Should I convert those to not include the prefix as part of this PR?